frontend/src/scenes/data-management/events/EventDefinitionsTable.tsx

```diff
@@ -20,22 +20,35 @@ export const scene: SceneExport = {
     paramsToProps: () => ({ syncWithUrl: true }),
 }
 
-interface EventDefinitionsTableProps {
-    compact?: boolean
-}
-
-export function EventDefinitionsTable({}: EventDefinitionsTableProps = {}): JSX.Element {
+export function EventDefinitionsTable(): JSX.Element {
     const { eventDefinitions, eventDefinitionsLoading, openedDefinitionId } = useValues(eventDefinitionsTableLogic)
-    const { loadEventDefinitions, setOpenedDefinition } = useActions(eventDefinitionsTableLogic)
+    const { loadEventDefinitions, setOpenedDefinition, setLocalEventDefinition } =
+        useActions(eventDefinitionsTableLogic)
     const { hasDashboardCollaboration, hasIngestionTaxonomy } = useValues(organizationLogic)
 
     const columns: LemonTableColumns<EventDefinition> = [
+        {
+            key: 'icon',
+            className: 'definition-column-icon',
+            render: function Render(_, definition: EventDefinition) {
+                return <EventDefinitionHeader definition={definition} hideView hideText />
+            },
+        },
         {
             title: 'Name',
             key: 'name',
             className: 'definition-column-name',
             render: function Render(_, definition: EventDefinition) {
-                return <EventDefinitionHeader definition={definition} hideView />
+                return (
+                    <EventDefinitionHeader
```

Comment:
I found myself clicking on this header to try to expand the event to see it's properties. Maybe it should be clickable?

Reply:
Hm I can see how that'd be more intuitive but I don't think we use that pattern anywhere else (i.e., Persons, Live events, etc.). Leaving this for another PR if we decided to change it app wide.